src/EnforcerManager.php

```diff
@@ -87,7 +87,8 @@ protected function resolve($name)
         }
 
         $model = new Model();
-        $loader = $this->app->make(ModelLoader::class, $config);
+        $loader = $this->app->make(LoaderManager::class);
+        $loader->initFromConfig($config);
```

Comment:
Please add config to the constructor for one-time initialization

Reply:
I have tried that, and I noticed that passing parameters every time an object is created leads to the repeated instantiation of the singleton instance. This not only violates the design principle of the singleton pattern but also causes the custom ModelLoader to fail when extending the class. Therefore, making the parameter passing optional can prevent these issues. Is there a superior solution?🤔

Reply:
I have tried that, but passing parameters through the constructor every time may lead to repeated instantiation of the singleton pattern and affect the extension of the custom ModelLoader. Is there a superior solution?🤔

Reply:
That makes sense, so keep it that way.